### PR TITLE
The implementation

### PR DESCRIPTION
**Pull request by smnbbrv**
Hi again,

just had some time to implement the things... Please check whether it is fine and let me know the result.

There are no tests yet, I think we can skip it for now; it will become important if there will be many changes / requests in future.

If everything is OK, please create an NPM package.

Cheers

### CODE REVIEW

**Comment by smnbbrv** [comment]
I think it also makes sort of sense to rename the repo to ng2-sortablejs. From the libraries I use everyone follows this naming notation...

File: README.md [hunk 1]
```diff
@@ -1,5 +1,119 @@
...
+First, include `SORTABLEJS_DIRECTIVES` into your component:
+
+    import { Component } from '@angular/core';
+    import { SORTABLEJS_DIRECTIVES } from 'angular-sortablejs';
+
+    @Component({
+        selector: 'my-app',
+        template: `
+          <h2>Drag / drop the item</h2>
+          <div [sortablejs]="items">
```
Comment (RubaXa) [inline README.md]: Is there a way not to specify `items`, as has been done before?

Comment (smnbbrv) [inline README.md]: I'm not really sure about that. Angular2 is quite different, at the moment I don't know the way.

Comment (RubaXa) [inline README.md]: Okay.

Comment (smnbbrv) [inline README.md]: @RubaXa actually I don't think it is a blocker problem at the moment. It does not bother that much to pass the array 2 times; furthermore it is easy to deprecate in future if we find the way to parse `ngFor`: we'll simply make the parameter optional.

**Comment by RubaXa** [comment]
@smnbbrv 
```
npm publish --registry=http://registry.npmjs.org/
+ angular-sortablejs@0.1.0
```

**Comment by RubaXa** [comment]
@smnbbrv Send me your npm@login

**Comment by smnbbrv** [comment]
@RubaXa awesome, https://www.npmjs.com/package/angular-sortablejs :+1: 

The NPM user is just the same, smnbbrv.

**Comment by RubaXa** [comment]
Done:
```
npm owner add smnbbrv --registry=http://registry.npmjs.org/
+ smnbbrv (angular-sortablejs)
```

**Pull request merged**